examples/live-blog-non-floating-button.amp.html

```diff
@@ -0,0 +1,254 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Lorem Ipsum | PublisherName</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Georgia|Open+Sans|Roboto' rel='stylesheet' type='text/css'>
+  <style amp-custom>
+    body {
+      margin: 0;
+      font-family: 'Georgia', Serif;
+    }
+
+    .brand-logo {
+      font-family: 'Open Sans';
+    }
+
+    .ad-container {
+      display: flex;
+      justify-content: center;
+    }
+
+    .content-container p {
+      line-height: 24px;
+    }
+
+    body > header {
+      background-color: #ff4464;
+      width: 100%;
+    }
+
+    .headline a {
+      color: black;
+      text-decoration: none;
+    }
+
+    header,
+    .headline,
+    .article-body {
+      padding: 15px;
+    }
+
+    figure {
+      margin: 0;
+    }
+
+    figcaption {
+      color: #6f757a;
+      padding: 15px 0;
+      font-size: .9em;
+    }
+
+    .author {
+      display: flex;
+      align-items: center;
+
+      background: #f4f4f4;
+      padding: 0 15px;
+
+      font-size: .8em;
+
+      border: solid #dcdcdc;
+      border-width: 1px 0;
+    }
+
+    .header-time {
+      color: #a8a3ae;
+      font-family: 'Roboto';
+      font-size: 12px;
+    }
+
+    .author p {
+      margin: 5px;
+    }
+
+    .byline {
+      font-family: 'Roboto';
+      display: inline-block;
+    }
+
+    .byline p {
+      line-height: normal;
+    }
+
+    .byline .brand {
+      color: #6f757a;
+    }
+
+    .standfirst {
+      color: #6f757a;
+    }
+
+    .mailto {
+      text-decoration: none;
+    }
+
+    #author-avatar {
+      margin: 10px;
+      border: 5px solid #fff;
+      width: 50px;
+      height: 50px;
+      border-radius: 50%;
+    }
+
+    h1 {
+      margin: 5px 0;
+      font-weight: normal;
+    }
+
+    footer {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      height: 226px;
+      background: #f4f4f4;
+    }
+
+    hr {
+      margin: 0;
+    }
+
+    amp-img {
+      background-color: #f4f4f4;
+    }
+
+    .slot-fallback {
+      padding: 16px;
+      background: yellow;
+    }
+
+    .amp-live-list-item {
+      border: 1px solid #dcdcdc;
+      border-width: 1px 0 0;
+    }
+
+    .social-box {
+      text-align: right;
+      margin: 10px 0;
+    }
+
+    .update-panel {
+      background-color: #f4f4f4;
+      border: 1px #dcdcdc solid;
+      height: 40px;
+      cursor: pointer;
+    }
+
+    .update-panel:hover {
+      background-color: #dcdcdc;
+    }
+
+    #live-blog-1 > .update-panel.amp-active {
```

Comment:
@kul3r4 @ericlindley-g fyi, added demo which has a more twitter like button that slides down instead of a fixed position floating button